platform/ios/jazzy.yml

```diff
@@ -23,53 +26,65 @@ custom_categories:
       - MGLMapViewDelegate
       - MGLStyle
       - MGLUserTrackingMode
-  - name: Annotations
+  - name: Protocols and Abstract Classes
     children:
       - MGLAnnotation
-      - MGLAnnotationImage
-      - MGLAnnotationView
-      - MGLCalloutView
-      - MGLCalloutViewDelegate
+      - MGLOverlay
+      - MGLShape
       - MGLMultiPoint
-      - MGLMultiPolygon
-      - MGLMultiPolyline
+      - MGLFeature
+  - name: Primitive Shapes
+    children:
       - MGLPointAnnotation
       - MGLPointCollection
       - MGLPolygon
       - MGLPolyline
-      - MGLOverlay
-      - MGLShape
+      - MGLMultiPolygon
+      - MGLMultiPolyline
       - MGLShapeCollection
+  - name: Annotations
+    children:
+      - MGLAnnotationImage
+      - MGLAnnotationView
       - MGLUserLocation
       - MGLUserLocationAnnotationView
-  - name: Map Data
+  - name: User Interaction
+    children:
+      - MGLCalloutView
+      - MGLCalloutViewDelegate
+  - name: Style Features
     children:
-      - MGLFeature
-      - MGLMultiPolygonFeature
-      - MGLMultiPolylineFeature
-      - MGLPointCollectionFeature      
       - MGLPointFeature
       - MGLPolygonFeature
       - MGLPolylineFeature
+      - MGLMultiPolygonFeature
+      - MGLMultiPolylineFeature
+      - MGLPointCollectionFeature
       - MGLShapeCollectionFeature
+  - name: Style Sources
+    children:
+      - MGLSource
+      - MGLTileSet
+      - MGLShapeSource
+      - MGLRasterSource
+      - MGLVectorSource
+  - name: Style Layers (Abstract)
+    children:
+      - MGLStyleLayer
+      - MGLForegroundStyleLayer
+      - MGLVectorStyleLayer
   - name: Style Layers
     children:
       - MGLBackgroundStyleLayer
+      - MGLRasterStyleLayer
       - MGLCircleStyleLayer
       - MGLFillStyleLayer
-      - MGLForegroundStyleLayer
       - MGLLineStyleLayer
-      - MGLRasterStyleLayer
-      - MGLStyleLayer
       - MGLSymbolStyleLayer
-      - MGLVectorStyleLayer
-  - name: Data Sources
+  - name: Style Values
     children:
-      - MGLSource
-      - MGLShapeSource
-      - MGLRasterSource
-      - MGLTileSet
-      - MGLVectorSource
+      - MGLStyleValue
+      - NSValue(MGLAdditions)
```

Comment:
Style value–related methods are only part of this category; there are also methods related to offline maps.